bin/dependency-visualizer

```diff
@@ -21,13 +22,19 @@ list($options, $file) = parseArgs($argv);
 if ($options['help']) {
     echo 'Usage: dependency-visualizer [options] <target php file>' . PHP_EOL;
     echo 'Options:' . PHP_EOL;
+    echo '  -d: Load Xdebug.' . PHP_EOL;
     echo '  -h: Show this help.' . PHP_EOL;
     echo '  -l: Output log to current directory.' . PHP_EOL;
     echo '  -m: Draw methods in diagram.' . PHP_EOL;
     echo '  -s: Output analysis results in string format.' . PHP_EOL;
     exit(0);
 }
 
+if ($options['loadXdebug'] === false) {
+    $xdebug = new XdebugHandler('dependency-visualizer');
+    $xdebug->check();
+    unset($xdebug);
+}
```

Comment:
> which means Xdebug is loaded by default unless explicitly disabled. 

No, I don't agree with the idea. Default value is set as false in parseArgs() function and it can be set to true only when  the -d opition is choosen. Have a different opinion?

Reply:
No worries at all, and thank you for reviewing.